.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish package to PyPi
```

Comment:
i don't think we need this file. as we're not going to publish this to PyPi (cause it's a service, and not a package)